tasks/cockpit-tasks-webhook.yaml

```diff
@@ -149,4 +149,4 @@ items:
         auth_mechanisms.1 = EXTERNAL
         ssl_cert_login_from = common_name
         management.load_definitions = /etc/rabbitmq/definitions.json
-        heartbeat = 360
+        heartbeat = 720
```

Comment:
It seems a bit weird to change it on this end -- don't the heartbeats from the clients come through? If they do, but are sent too rarely, then it would be better to increase their frequency on the client side. If they don't, we should disable them completely (value 0).

But, this won't break anything, so let's try that.